Scala/Scala.sublime-syntax

```diff
@@ -569,6 +569,11 @@ contexts:
   class-type-parameter-list:
     - match: '\b(private|protected)\b'
       scope: storage.modifier.access.scala
+    - match: (?=@{{plainid}})
+      set:
+        - include: annotation
+        - match: '(?=\S)'
```

Comment:
Can you add a few tests which cover this? The following should be good starting points:

```scala
class Foo @bar (x: Int)
class Foo @bar(42) [A](x: Int)
class Foo @bar @baz (x: Int)
```

Reply:
Ok. I will add them as soon I can.